modules/contact/contact.service.js

```diff
@@ -43,13 +43,17 @@ class ContactService {
 
     if (!contact) return null;
 
-    return contact.images.length === 2 &&
+    if (
+      contact.images.length === 2 &&
       data.mapImages.length === 2 &&
       this.deleteMapImages(contact)
-      ? await this.saveUpdatedContact(data)
-      : await Contact.findByIdAndUpdate(data.id, data.contact, {
-          new: true,
-        }).exec();
+    ) {
+      await this.saveUpdatedContact(data);
+    } else {
+      await Contact.findByIdAndUpdate(data.id, data.contact, {
```

Comment:
Could you add "return" word to if-else statement to return something to the front-end

Reply:
sure I can do this, but it will change the logic, and it is not related to code smell task.
I think we can create a special task for it to have scope for check and testing regressions